### DISCUSSION TITLE
Duplicate requests from LittleProxy

### DISCUSSION THREAD

**Discussion by Au6ojlut** [selenide/selenide#2280] [Q&A]
I'm almost sure, that the problem is not related to Selenide, but maybe someone is faced the same problem:

In our project we use SelenideProxy to redirect some requests based on some rule and sometimes there is a duplicate request (!) that is fired from the `LittleProxy` when **next** UI test already in progress (!).

I'm not familiar with the `LittleProxy`, so the question is: 

`Is it possible to reset proxy or check does it still has some requests to fire?`

**Comment by asolntsev** [comment]
@Au6ojlut If I understand correctly, the problem is not with proxy. Proxy doesn't "hold" any requests, proxy doesn't "fire" any requests. Proxy only gets requests from browser and forwards them to server.

What you really need is resetting browser, not proxy. Be sure the browser has completed all previous requests before starting a new test. 

See https://www.youtube.com/watch?v=ePvrXUCeAr8&ab_channel=SeleniumCamp&t=38m25s
